### ISSUE TITLE
Running multiple benchmarks hangs

### ISSUE THREAD

**Issue by haf** [haf/expecto#246]
See https://github.com/dotnet/BenchmarkDotNet/issues/755 for details

Not sure on if this is in Expecto or in BenchmarkDotnet.

**Comment by haf** [comment]
I don't think so; if you look towards the end of Program.fs you can see I'm running it sequenced already.

**Comment by AnthonyLloyd** [comment]
Then I guess its an issue with BenchmarkDotNet. As long as we are really running sequenced and are cleaning up ok.
